backend/spec/models/survey_spec.rb

```diff
@@ -50,4 +50,22 @@
       expect(survey).to be_valid
     end
   end
+
+  describe '#answered_questions_quantity' do
+    subject(:answered_quantity) { survey.answered_questions_quantity }
+
+    context 'when there is 1 answer' do
+      let(:user) { survey.answers_surveys.first.user }
+      let(:survey) { create(:survey_with_answer) }
```

Comment:
The order doesn't matter here? This is new for me. 😅 

```suggestion
      let(:user) { survey.answers_surveys.first.user }
      let(:survey) { create(:survey_with_answer) }
```
```suggestion
      let(:survey) { create(:survey_with_answer) }
      let(:user) { survey.answers_surveys.first.user }
```